test/NetCore.Test/NetCore.Test.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We probably want to run this in CI. That means adding it to `Test.cmd`
We might also want to include this in the linux build (at least building, and eventually running the tests too)

Reply:
Alright, I gave it a go so let's see what happens.